data/WOTC_5e_SRD_v5.1/armor.json

```diff
@@ -0,0 +1,187 @@
+      "cost": "10 gp",
+      "weight": "10 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "studded_leather_armour",
+      "name": "Studded Leather Armor",
+      "type": "Light Armor",
+      "rarity": "Standard",
+      "ac": 12,
+      "cost": "45 gp",
+      "weight": "13 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "hide",
+      "name": "Hide",
+      "type": "Medium Armor",
+      "ac": 12,
+      "cost": "10 gp",
+      "weight": "12 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "chain_shirt",
+      "name": "Chain Shirt",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 13,
+      "cost": "50 gp",
+      "weight": "20 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "scale",
+      "name": "Scale mail",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 14,
+      "cost": "50 gp",
+      "weight": "45 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": true
+    }, {
+      "id": "breastplate",
+      "name": "Breastplate",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 14,
+      "cost": "400 gp",
+      "weight": "20 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": false
+    }, {
+      "id": "half_plate",
+      "name": "Half plate",
+      "type": "Medium Armor",
+      "rarity": "Standard",
+      "ac": 15,
+      "cost": "750 gp",
+      "weight": "40 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": true
+    }, {
+      "id": "ring_mail",
+      "name": "Ring mail",
+      "type": "Heavy Armor",
+      "rarity": "Standard",
+      "ac": 14,
+      "cost": "30 gp",
+      "weight": "40 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": {},
+      "disadvantage": true
+    }, {
+      "id": "chain_mail",
+      "name": "Chain mail",
+      "type": "Heavy Armor",
+      "rarity": "Standard",
+      "ac": 16,
+      "cost": "75 gp",
+      "weight": "55 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": { "strength": 13 },
+      "disadvantage": true
+    }, {
+      "id": "splint",
+      "name": "Splint",
+      "type": "Heavy Armor",
+      "rarity": "Standard",
+      "ac": 17,
+      "cost": "200 gp",
+      "weight": "60 lb.",
+      "modifier": [ "dexterity" ],
+      "requirements": { "strength": 15 },
+      "disadvantage": true
+    }, {
+      "id": "plate_mail",
+      "name": "Plate Mail",
+      "type": "Heavy Armor",
+      "rarity": "Standard",
+      "ac": 18,
+      "cost": "1500 gp",
+      "weight": "65 lb.",
+      "requirements": { "strength": 15 },
+      "modifier": [ "dexterity" ],
```

Comment:
Similar to the medium armor, shouldn't this be blank? According to the rules Plate doesn't add any bonus.

Reply:
good catch